blazar/plugins/devices/k3s_plugin.py

```diff
@@ -148,14 +153,20 @@ def poll_resource_failures(self, devices):
         return failed_devices, recovered_devices
 
     def allocate(self, device_reservation, lease, devices):
+        reservation = db_api.reservation_get(
+            device_reservation["reservation_id"])
+        lease = db_api.lease_get(reservation["lease_id"])
+        project_id = lease["project_id"]
```

Comment:
Can you not get this on the `lease` that is on the function signature?

Reply:
🤦